exchange/utils_test.go

```diff
@@ -4769,3 +4769,173 @@ func TestApplyBidAdjustmentToFloor(t *testing.T) {
 		})
 	}
 }
+
+func TestBuildRequestExtAlternateBidderCodes(t *testing.T) {
+	type testInput struct {
+		bidderNameRaw string
+		accABC        *openrtb_ext.ExtAlternateBidderCodes
+		reqABC        *openrtb_ext.ExtAlternateBidderCodes
+	}
+	testCases := []struct {
+		desc     string
+		in       testInput
+		expected *openrtb_ext.ExtAlternateBidderCodes
+	}{
+		{
+			desc:     "No biddername",
+			in:       testInput{},
+			expected: nil,
+		},
+		{
+			desc: "non-nil reqABC",
+			in: testInput{
+				bidderNameRaw: "pubmatic",
+				reqABC:        &openrtb_ext.ExtAlternateBidderCodes{},
+			},
+			expected: &openrtb_ext.ExtAlternateBidderCodes{},
+		},
+		{
+			desc: "non-nil reqABC",
```

Comment:
Can you either update this test case or add another test case so that the returned object contains alternate bidder codes from the account config? (request config must be empty in this case)